config/log-viewer.php

```diff
@@ -2,6 +2,27 @@
 
 use Opcodes\LogViewer\LogLevels\LevelClass;
 
+if (!function_exists('renv')) {
+	function renv(string $cst, ?string $default = null): string
+	{
+		return rtrim(env($cst, $default) ?? '', '/');
+	}
+}
+
+/**
+ * Allow to conditionally append an env value.
+ *
+ * @param string $cst constant to fetch
+ *
+ * @return string '' or env value postfixed with '/'
```

Comment:
```suggestion
 * @return string '' or env value prefixed with '/'
```

Is there no way to avoid repeating these functions?

Reply:
I wish and tried. But it was being annoying with composer loading etc.